test/common/namespace_created.go

```diff
@@ -3,6 +3,9 @@ package common
 import (
 	goctx "context"
 	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	pkgresources "github.com/integr8ly/integreatly-operator/pkg/resources"
+	configv1 "github.com/openshift/api/config/v1"
+	"golang.org/x/net/context"
```

Comment:
```suggestion
	goctx "context"
	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
	pkgresources "github.com/integr8ly/integreatly-operator/pkg/resources"
	configv1 "github.com/openshift/api/config/v1"
```

Could we use `goctx "context"` instead of importing `"golang.org/x/net/context"` to keep the use of `context` consistent ?

Reply:
@KevFan Thank you for comments/suggestions.   It's done. Thanks!